src/libraries/System.Runtime.Intrinsics/tests/Vectors/Vector128Tests.cs

```diff
@@ -1,13 +1,3820 @@
+
+                for (int index = 0; index < Vector128<byte>.Count; index++)
+                {
+                    Assert.Equal((byte)0x1, value[index]);
+                }
+            }
+            finally
+            {
+                NativeMemory.AlignedFree(value);
+            }
+        }
+
+        [Fact]
+        public unsafe void Vector128DoubleStoreAlignedNonTemporalTest()
+        {
+            double* value = null;
+
+            try
+            {
+                value = (double*)NativeMemory.AlignedAlloc(byteCount: 16, alignment: 16);
+
+                value[0] = 0;
+                value[1] = 1;
+
+                Vector128.Create((double)0x1).StoreAlignedNonTemporal(value);
+
+                for (int index = 0; index < Vector128<double>.Count; index++)
+                {
+                    Assert.Equal((double)0x1, value[index]);
+                }
+            }
+            finally
+            {
+                NativeMemory.AlignedFree(value);
+            }
+        }
+
+        [Fact]
+        public unsafe void Vector128Int16StoreAlignedNonTemporalTest()
+        {
+            short* value = null;
+
+            try
+            {
+                value = (short*)NativeMemory.AlignedAlloc(byteCount: 16, alignment: 16);
+
+                value[0] = 0;
+                value[1] = 1;
+                value[2] = 2;
+                value[3] = 3;
+                value[4] = 4;
+                value[5] = 5;
+                value[6] = 6;
+                value[7] = 7;
+
+                Vector128.Create((short)0x1).StoreAlignedNonTemporal(value);
+
+                for (int index = 0; index < Vector128<short>.Count; index++)
+                {
+                    Assert.Equal((short)0x1, value[index]);
+                }
+            }
+            finally
+            {
+                NativeMemory.AlignedFree(value);
+            }
+        }
+
+        [Fact]
+        public unsafe void Vector128Int32StoreAlignedNonTemporalTest()
+        {
+            int* value = null;
+
+            try
+            {
+                value = (int*)NativeMemory.AlignedAlloc(byteCount: 16, alignment: 16);
+
+                value[0] = 0;
+                value[1] = 1;
+                value[2] = 2;
+                value[3] = 3;
+
+                Vector128.Create((int)0x1).StoreAlignedNonTemporal(value);
+
+                for (int index = 0; index < Vector128<int>.Count; index++)
+                {
+                    Assert.Equal((int)0x1, value[index]);
+                }
+            }
+            finally
+            {
+                NativeMemory.AlignedFree(value);
+            }
+        }
+
+        [Fact]
+        public unsafe void Vector128Int64StoreAlignedNonTemporalTest()
+        {
+            long* value = null;
+
+            try
+            {
+                value = (long*)NativeMemory.AlignedAlloc(byteCount: 16, alignment: 16);
```

Comment:
Any reason not to condense it to:
```suggestion
            long* value = (long*)NativeMemory.AlignedAlloc(byteCount: 16, alignment: 16);
            try
            {
```
?

Reply:
I have a habit of putting the thing that is impacted by the `catch` in the `try` to differentiate on what failed and avoid issues with `SkipLocalsInit`, etc.